classes/controller/AdminController.php

```diff
@@ -2145,7 +2150,12 @@ protected function addPageHeaderToolBarModulesListButton()
     {
         $this->filterTabModuleList();
 
-        if (is_array($this->tab_modules_list['slider_list']) && count($this->tab_modules_list['slider_list'])) {
+        /**
+         * @TODO remove isInstalled when MBO module will be released
+         */
+        if (!Module::isInstalled('ps_mbo') &&
+            is_array($this->tab_modules_list['slider_list']) &&
+            count($this->tab_modules_list['slider_list'])) {
```

Comment:
When writing multiline ifs, please put the closing parenthesis on a new line, like so:

```php
if ($foo
    && $bar
    && $baz
) {
    doDoge();
}

Reply:
Woot, how can I did that :O